src/api/tests/test_user_course.py

```diff
@@ -0,0 +1,40 @@
+from .util import Client
```

Comment:
Please add one more test case for the 'Not Authorized' case.
This should be pretty straight forward, you just need to try to post a new course without logging in (calling session post) and assert you get 403 in response.